Makefile

```diff
@@ -140,6 +140,9 @@ run: generate manifests
 install: manifests
 	kustomize build config/crd | kubectl apply -f -
 
+deploy-mutation: deploy
+	kustomize build --load_restrictor LoadRestrictionsNone config/overlays/mutation | kubectl apply -f -
```

Comment:
Not a big fan of removing load restrictions, but it seems like this is probably the cleanest way of  having dev CRDs, otherwise we'd need to change the file structure a bit, which may not play well with controller-gen.

TL;DR this works